docs/RFCS/online_schema_change.md

```diff
@@ -131,8 +131,15 @@ Since schema change operations are potentially long running they need
 to be restartable or abortable if the node performing them dies. We
 accomplish this by performing the schema change operation for a table
 on a well known node: the replica holding the leader lease for the
-first range of the table (i.e. containing the key `/<tableID>`). When
-a node receives a schema change operation such as `CREATE INDEX` it
+first range of the table (i.e. containing the key `/<tableID>`), known
+as the "table leader". Each node discovers via gossip the list of
+all configured tables, and starts a goroutine to execute schema
```

Comment:
I'm not sure the table leader will start a goroutine to handle schema changes when it becomes leader. This might very well be some sort of callback system `Leader{Start,End}` or it might be a `LeaderStart` callback that takes a channel which is closed when leadership is lost.